docs/topics/appendix-s2i-build-process.adoc

```diff
@@ -1,16 +1,16 @@
 [[s2i-build-process]]
 = The Source-to-Image (S2I) Build Process
-link:https://docs.openshift.com/container-platform/3.4/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image] (S2I) is a build tool for generating reproducible Docker-formatted container images from online SCM repositories with application sources.
+link:https://docs.openshift.com/container-platform/latest/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image] (S2I) is a build tool for generating reproducible Docker-formatted container images from online SCM repositories with application sources.
 With S2I builds, you can easily deliver the latest version of your application into production with shorter build times, decreased resource and network usage, improved security, and a number of other advantages.
-For more information, see the link:https://docs.openshift.com/container-platform/3.4/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image (S2I) Build] chapter of the OpenShift Container Platform documentation.
+For more information, see the link:https://docs.openshift.com/container-platform/latest/architecture/core_concepts/builds_and_image_streams.html#source-build[Source-to-Image (S2I) Build] chapter of the OpenShift Container Platform documentation.
 
 You must provide three elements to the S2I process:
 
 * The application sources hosted in an online SCM repository, such as GitHub.
 
-* The link:https://docs.openshift.com/container-platform/3.4/creating_images/s2i.html#s2i-scripts[S2I scripts].
+* The link:https://docs.openshift.com/container-platform/latest/creating_images/s2i.html[S2I scripts].
 
```

Comment:
to assemble the final Docker image

Reply:
@cmoulliard  --  will use" assemble the final container image".